platform/nodejs/eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "env": {
+        "es2021": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
+        "indent": [ "warn", 4 ],
```

Comment:
This is a _most basic_ configuration. If there are indentation issues, the linter will give a warning. Later this should be updated so that it gives errors.